### PR TITLE
Maze of mirrors compliance

### PR DESCRIPTION
**Pull request by mgirman520**
using base.IsVillain(c) as opposed to c.IsVillain on hero cards so that Maze of Mirrors can correctly intercept them.

### CODE REVIEW

File: Controller/Heroes/Starlight/Cards/WarpHaloCardController.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ public class WarpHaloCardController : StarlightCardController
         public WarpHaloCardController(Card card, TurnTakerController turnTakerController) : base(card, turnTakerController)
         {
             SpecialStringMaker.ShowListOfCards(new LinqCardCriteria((Card c) => c.IsTarget && c.IsHero && IsNextToConstellation(c), useCardsSuffix: false, singular: "hero target next to a constellation", plural: "hero targets next to a constellation"));
-            SpecialStringMaker.ShowListOfCards(new LinqCardCriteria((Card c) => c.IsTarget && c.IsVillain && IsNextToConstellation(c), useCardsSuffix: false, singular: "villain target next to a constellation", plural: "villain targets next to a constellation"));
+            SpecialStringMaker.ShowListOfCards(new LinqCardCriteria((Card c) => c.IsTarget && base.IsVillain(c) && IsNextToConstellation(c), useCardsSuffix: false, singular: "villain target next to a constellation", plural: "villain targets next to a constellation"));
```
Comment (LNBertolini) [inline Controller/Heroes/Starlight/Cards/WarpHaloCardController.cs:13]: `            SpecialStringMaker.ShowListOfCards(new LinqCardCriteria((Card c) => c.IsTarget && !c.IsHero && IsNextToConstellation(c), useCardsSuffix: false, singular: "non-hero target next to a constellation", plural: "non-hero targets next to a constellation"));
`

This isn't a mistake in the effect, it's a mistake in the SpecialString

**Comment by LNBertolini** [comment]
There's another one on Doc Havoc's Immediate Evac ("each villain target regains 2 HP")

**Comment by Vilnor** [comment]
Looks like there also a base.IsVillainTarget to use instead of c.IsVillainTarget

File: Controller/Heroes/Starlight/Cards/WarpHaloCardController.cs (hunk 1 above)
Comment (Vilnor) [reply Controller/Heroes/Starlight/Cards/WarpHaloCardController.cs]: Committing correction

**Review by LNBertolini — APPROVED** [review]

**Pull request merged**